### ISSUE TITLE
[Bug] Sharing screen not work in MacOS

### ISSUE THREAD

**Issue by kidclone3** [SpikeHD/Dorion#270]
**Describe the bug**
When I join a channel and click the "Share Your Screen" button, nothing happend.

**System Info (please complete the following information):**
 - OS: MacOS Sonoma 14.7.1 (latest of Sonoma)
 - Version: 6.2.0
 - Extra Client Mods/Plugins: No Mods

**Comment by SpikeHD** [comment]
This sounds like it's related to https://github.com/tauri-apps/tauri/issues/10878, which itself it related to https://github.com/tauri-apps/wry/issues/1195 and this PR: https://github.com/tauri-apps/wry/pull/1196.

As far as I can tell (and I could be wrong, it's a little hard to follow the thread of all the different issues and such lol), the main library Dorion uses is suppressing, and therefore auto-denying, the screenshare permission window, so we'll have to wait for that to get fixed up. This is also occurring on my 14.1 machine.

**Comment by LewsTherinTelescope** [comment]
It works for me now, with the caveat that like normal browser Discord and most third-party clients it doesn't support sharing audio. [Legcord did recently get that working](https://github.com/Legcord/Legcord/pull/1092), but they have their own screen picker and the PR uses some Chromium-specific flags, so I'm not sure if it's applicable to Dorion or not.

**Comment by SpikeHD** [comment]
They are not, glad it works now though!

**Comment by SocietasEvanescentes** [comment]
Hello! I'm using v1.3.0, but I'm still facing the issue of audio not being shared. 

Is there any way to share my apps diagnostic information to see what could be causing this still? 😅

**Comment by SpikeHD** [comment]
Audio being shared is a separate issue, feel free to make a bug report 👍 

(also 1.3.0 is an extremely old version, you should definitely update)
